addons/web_editor/static/tests/field_html_tests.js

```diff
@@ -148,9 +148,7 @@ QUnit.module('web_editor', {}, function () {
             testUtils.mock.intercept(form, 'call_service', function (ev) {
                 if (ev.data.service === 'notification') {
                     assert.deepEqual(ev.data.args[0], {
-                        "className": undefined,
                         "message": "<ul><li>Header</li></ul>",
```

Comment:
Probably requires `isMessageHtml`?

Reply:
Good catch, either that or again an example where it's just a list.